questionnaire/questionnaire-dal.js

```diff
@@ -118,12 +118,36 @@ function questionnaireDAL(spec) {
         return result;
     }
 
+    async function getQuestionnaireIdsBySubmissionStatus(submissionStatus) {
+        let result;
+
+        try {
+            result = await db.query('SELECT id FROM questionnaire WHERE submission_status = $1', [
+                submissionStatus
+            ]);
+
+            if (result.rowCount === 0) {
+                throw new VError(
```

Comment:
I'd expect something that selects by status to return 0 or more things, should this be an exception?